packages/components/src/Chips/ChipsTypes.tsx

```diff
@@ -0,0 +1,60 @@
+import { ReactElement } from "react";
+import { XOR } from "ts-xor";
+import { ChipProps } from "./Chip";
+
+interface ChipFoundationProps {
+  readonly children: ReactElement<ChipProps>[];
+
+  /**
+   * Change the interaction on the chip.
+   */
+  readonly type?: "singleselect" | "multiselect";
+
+  /**
+   * Determines which chip gets highlighted.
+   *
+   * The type of the value depends on what you pass in as the selected chips or
+   * the value of the chip child.
+   */
+  readonly selected?: string | string[];
+
+  /**
+   * Callback whenever a chip is clicked. This returns a new value of
+   * selected chips.
+   *
+   * @param value
+   */
+  onChange(value?: string | string[]): void;
+
+  /**
+   * Callback when a specific chip is clicked
+   *
+   * @param event
+   * @param clickedChipValue - The value of the chip that was clicked
+   */
+  onClickChip?(
```

Comment:
can we rename this to `onClick`?

Reply:
related [this comment](https://github.com/GetJobber/atlantis/pull/678#discussion_r718867899) to answer [here](https://github.com/GetJobber/atlantis/pull/678#discussion_r718919839)